ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/machine/WsAgentStateController.java

```diff
@@ -41,40 +46,49 @@
 import java.util.List;
 
 import static com.google.common.collect.Lists.newArrayList;
+import static org.eclipse.che.api.core.model.workspace.WorkspaceStatus.RUNNING;
+import static org.eclipse.che.ide.MimeType.APPLICATION_JSON;
 import static org.eclipse.che.ide.api.machine.WsAgentState.STARTED;
 import static org.eclipse.che.ide.api.machine.WsAgentState.STOPPED;
+import static org.eclipse.che.ide.rest.HTTPHeader.ACCEPT;
 
 /**
  * @author Roman Nikitenko
  * @author Valeriy Svydenko
  */
 @Singleton
```

Comment:
No single word about purpose of this class, the same for public methods.
Please add java docs.